force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -161,8 +180,8 @@ public with sharing class VariablesDataTypesOperators {
      */
     Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer milliseconds;
```

Comment:
Try to use the MILLISECONDS_PER_MINUTE by make it static, Final and assigning it 60000